### PR TITLE
update to EPSG v9.8.13 database

### PR DESCRIPTION
**Pull request by jmckenna**
- update database to EPSG v9.8.13   released 2020-07-01
 - this was generated with the EPSG v9 series model

### CODE REVIEW

File: data/sql/unit_of_measure.sql [hunk 1]
```diff
@@ -3,95 +3,95 @@
 INSERT INTO "unit_of_measure" VALUES('EPSG','1024','bin','scale',1.0,NULL,0);
 INSERT INTO "unit_of_measure" VALUES('EPSG','1025','millimetre','length',0.001,NULL,0);
 INSERT INTO "unit_of_measure" VALUES('EPSG','1026','metre per second','length',1.0,NULL,0);
-INSERT INTO "unit_of_measure" VALUES('EPSG','1027','millimetres per year','length',3.16887651727314875889e-11,NULL,0);
+INSERT INTO "unit_of_measure" VALUES('EPSG','1027','millimetres per year','length',3.16887651727314922922e-11,NULL,0);
```
Comment (jjimenezshaw) [inline data/sql/unit_of_measure.sql:6]: Looks like all floating point values with lots of decimals changed (not for 'US survey foot').
Just curiosity: did it actually change in the EPSG db, or is it due to any operation in our side?

**Comment by jjimenezshaw** [comment]
@jmckenna I think I know why building is failing. I created a PR over your branch: https://github.com/jmckenna/PROJ/pull/1
One of the new entries in EPSG is `EPSG:9410` 'ETRS89 to Alicante height' that was already in `grid_transformation_custom.sql`.

**Comment by rouault** [comment]
data/sql/metadata.sql should also be manually updated

File: data/sql/unit_of_measure.sql (hunk 1 above)
Comment (jmckenna) [reply data/sql/unit_of_measure.sql:6]: I did use Python3, not sure if that is a factor.

**Comment by rouault** [comment]
Superseded by #2309 that updates to 9.8.15.

I couldn't reproduce the changes in unit of measures. I tried with Python 3.5 and 3.9.0a5. I also tried with different SQLite versions. So they are back to their original values in #2309

**Comment by jmckenna** [comment]
My environment is: Python 3.7.8     SQLite 3.32.3 (released 2020-06-18)

Will keep an eye on this for the v10 model.

**Pull request closed without merging**